### PR TITLE
Readme me and coc

### PR DESCRIPTION
**Pull request by mcrauwel**
#### Pull Request (PR) description

Bring docs up-to-date and include code of conduct to reference to VoxPupuli COC.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -51,28 +51,61 @@ You can customize options such as (but not limited to) `listen_port`, `admin_pas
...
+     mysql_users      => [ 
+       { 
+         'app' => { 
+           'password' => '*92C74DFBDA5D60ABD41EFD7EB0DAE389F4646ABB', 
+           'default_hostgroup' => 1, 
+         } 
+       },
+       { 
+         'ro'  => { 
+           'password' => mysql_password('MyReadOnlyUserPassword'),
```
Comment (bastelfreak) [inline README.markdown]: should those passwords be wrapped in ```Sensitive()```?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1 @@
+View the Vox Pupuli code of conduct here: https://voxpupuli.org/coc/
```
Comment (bastelfreak) [inline CODE_OF_CONDUCT.md]: should we add that to https://github.com/voxpupuli/modulesync_config/tree/master/moduleroot ?

Comment (mcrauwel) [reply CODE_OF_CONDUCT.md]: that would be a good idea IMHO

File: README.markdown (hunk 1 above)
Comment (mcrauwel) [reply README.markdown]: Good point, but should I need to update the type and provider code then also to deal with that?

Comment (bastelfreak) [reply README.markdown]: mhm I'm not sure, I think this is fine for now (and I should probably recheck how ```Sensitive()``` works)

**Pull request merged**